public/locales/de/translation.json

```diff
@@ -1,427 +1,439 @@
+    "about_this_app": "Über diese App",
+    "this_app_is_designed": "Diese App soll römischen Katholiken helfen, sich auf das Sakrament der Beichte vorzubereiten, indem sie ihr Gewissen prüfen. Sie ist <strong>NICHT</strong> ein Ersatz für die Beichte.",
+    "please_be_respectful": "Bitte nehmen Sie Rücksicht auf Ihre Mitmenschen, wenn Sie diese App nutzen. Ich empfehle Ihnen, Ihr Telefon auszuschalten, wenn Sie in Ihrer Kirche sind, und diese App zu benutzen, bevor Sie ankommen. Wenn Sie diese App in Ihrer Kirche oder während der Beichte verwenden, stellen Sie bitte sicher, dass Ihr Telefon auf lautlos gestellt ist.",
+    "this_website": "Diese Website, <website>ConfessIt.app</website>, basiert auf der <app>ConfessIt Android App</app> (2012 vom selben Entwickler erstellt). Es handelt sich zwar (noch) nicht um eine vollständige Reproduktion, aber sie soll die App einer größeren Anzahl von Nutzern auf einer breiteren Palette von Geräten zugänglich machen. (Diese Website funktioniert auf iOS, Android, Tablets und Computern!)",
+    "if_you_find": "Wenn du diese App nützlich findest, erwäge bitte, sie <strong>mit deinen Freunden und deiner Familie zu teilen</strong>. Erzählen Sie auf Facebook, Reddit, Twitter, in Ihrer Kirche oder in Ihrer Bibelstudiengruppe davon, um sie zu verbreiten!",
+    "privacy": "Datenschutz",
+    "information_you_enter": "Information you enter into this app is only stored on your device. It is not sent over the internet. We are able to do this using a technology provided by your web browser called <a>local storage</a>. We do not run Google Analytics or any other data collection mechanism on this site. Data you enter will be saved on your device until you hit <kbd>Clear</kbd> even if you close the window or refresh the page.",
+    "open_source": "Open Source",
+    "confessit_is_open_source": "ConfessIt ist Open Source. Wir entwickeln die App auf <i></i> <github>GitHub</github> und wir arbeiten in der <osc>Open Source Catholic</osc> Community auf Slack zusammen.",
+    "can_i_help_with_translations": "Kann ich bei Übersetzungen helfen?",
+    "confessit_is_translated": "ConfessIt is translated into multiple languages. If you'd like to help with this effort by adding a new translation or improving an existing translation, please read about how to do so on <github>GitHub</github> or get in touch with us on <osc>Open Source Catholic</osc> Slack.",
+    "can_i_help_write_code": "Kann ich beim Programmieren helfen?",
+    "we_welcome_new_contributions": "Wir begrüßen neue Beiträge. Wenn Sie zu ConfessIt beitragen möchten, lesen Sie am besten auf <1>GitHub</1> nach, wie Sie dies tun können.",
+    "about_the_developer": "",
+    "mike_kasberg": "<0>Mike Kasberg</0> develops ConfessIt in his free time, as a way of giving back to the church. He is also involved in a few other small projects to support Catholic organizations with technology.",
```

Comment:
`<0>` here seems wrong, in the English translation it's `<website>`. I want to avoid the use of indexed components in these strings since they've caused problems before: https://github.com/kas-catholic/confessit-web/pull/40

Reply:
of course it's going to be wrong in the other languages: it was wrong in the English source translation! This PR fixes the English. Weblate will take care of letting translators know that the translation no longer corresponds with the English source (this is known as a `dirty` translation). The other languages can very easily be fixed in Weblate.